src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -144,38 +144,37 @@ public UserManagementService(WorkOSClient client)
         /// Creates an email verification challenge and emails verification token to user.
         /// </summary>
         /// <param name="id">The unique ID of the User.</param>
-        /// <param name="options"> Parameters used to create email verification challenge.</param>
         /// <param name="cancellationToken">An optional token to cancel the request.</param>
         /// <returns> A token and the corresponding User object.</returns>
-        public async Task<(string, User)> CreateEmailVerificationChallenge(
+        public async Task<User> SendEmailVerificationEmail(
             string id,
-            CreateEmailVerificationChallengeOptions options,
             CancellationToken cancellationToken = default)
         {
             var request = new WorkOSRequest
             {
-                Options = options,
                 Method = HttpMethod.Post,
-                Path = $"/users/{id}/email_verification_challenge",
+                Path = $"/users/{id}/send_verification_email",
             };
-            return await this.Client.MakeAPIRequest<(string, User)>(request, cancellationToken);
+            return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
```

Comment:
This endpoint returns the user nested under the `user` field, so we'll need to update the response DTO to reflect that.